app/views/internal/feedback_messages/_abuse_reports.html.erb

```diff
@@ -44,26 +44,61 @@
             <span>
               <% if params[:status] == "Open" || params[:status].blank? %>
                 <% if reaction.reactable_type == "User" %>
-                  <%= form_for [:internal, reaction], html: { class: "d-inline" } do |f| %>
-                    <%= f.hidden_field :status, value: "confirmed" %>
-                    <%= f.submit "CONFIRMED", class: "btn btn-success btn-sm", data: { confirm: "Are you sure?" } %>
-                  <% end %>
+                  <button class="btn btn-success btn-sm js-reaction-confirm" type="button" data-reactable="user" data-id="<%= reaction.id %>" data-status="confirmed">CONFIRM</button>
                 <% else %>
-                  <%= form_for [:internal, reaction], html: { class: "d-inline" } do |f| %>
-                    <%= f.hidden_field :status, value: "confirmed" %>
-                    <%= f.submit "CONFIRMED", class: "btn btn-success btn-sm" %>
-                  <% end %>
-                <% end %>
-                <%= form_for [:internal, reaction], html: { class: "d-inline" } do |f| %>
-                  <%= f.hidden_field :status, value: "invalid" %>
-                  <%= f.submit "INVALID", class: "btn btn-danger btn-sm" %>
+                  <button class="btn btn-success btn-sm js-reaction-confirm" type="button" data-reactable="not-user" data-id="<%= reaction.id %>" data-status="confirmed">CONFIRM</button>
                 <% end %>
+                <button class="btn btn-danger btn-sm js-reaction-confirm" type="button" data-id="<%= reaction.id %>" data-status="invalid">INVALID</button>
               <% end %>
             </span>
           </div>
-          <hr>
+          <hr id="js__reaction__div__hr__<%= reaction.id %>">
         <% end %>
       </div>
     </div>
   </div>
 </div>
+
+<script>
+  function updateReactionStatus(id, status){
```

Comment:
you might want to explore https://docs.dev.to/internal/internal-user-interface/ to convert this block to a Stimulus controller, but definitely not a blocker for the PR :) Idea for the future ;)

Reply:
ok I will check it out